scripts/chain/start_chain.sh

```diff
@@ -120,10 +121,13 @@ else
 fi
 
 echo "INFO: Generating gentxs for validator accounts"
-for (( a=1; a<=$NUM_VALS; a++ ))
+for (( a=2; a<=$NUM_VALS; a++ ))
```

Comment:
why starting from 2?

Reply:
I've hardcoded the first one to generate the gravity-tx